profile.css

```diff
@@ -1 +1,69 @@
-// style goes here
+/* style goes here */
+:root{
```

Comment:
nice use of css variables.
though i recommend not using Uppercase case letters, a simple `--kabab-case` would be enough and make it consistent

Reply:
Will work on it.
Thank you for the feedback